pkg/agent/util/net.go

```diff
@@ -51,6 +52,32 @@ var (
 	netInterfaceAddrs   = (*net.Interface).Addrs
 )
 
+// Route stores Windows route item. In order to be able to read it when generating mocking code, define it in the file
```

Comment:
I agree with @antoninbas to introduce a subdirectory for Windows to resolve the mock requirement, rather than use define OS specific resources in a common file.